message/event.go

```diff
@@ -0,0 +1,123 @@
+package message
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/xmidt-org/wrp-go/v3"
+)
+
+const (
+	BootTimeKey = "/boot-time"
+)
+
+var (
+	ErrParseDeviceID    = errors.New("error getting device ID from event")
+	ErrBirthdateParse   = errors.New("unable to parse birthdate from payload")
+	ErrBootTimeParse    = errors.New("unable to parse boot-time")
+	ErrBootTimeNotFound = errors.New("boot-time not found")
+)
+
+// Event is the struct that contains the wrp.Message fields along with the birthdate
+// that is parsed from the payload.
+type Event struct {
+	MsgType         int               `json:"msg_type"`
+	Source          string            `json:"source"`
+	Destination     string            `json:"dest,omitempty"`
+	TransactionUUID string            `json:"transaction_uuid,omitempty"`
+	ContentType     string            `json:"content_type,omitempty"`
+	Metadata        map[string]string `json:"metadata"`
+	Payload         string            `json:"payload,omitempty"`
+	Birthdate       int64             `json:"birth_date"`
+	PartnerIDs      []string          `json:"partner_ids,omitempty"`
+}
+
+// NewEvent creates an Event from a wrp.Message and also parses the Birthdate from the
+// message payload, setting the birthdate to the defaultBirthdate if it cannot be parsed.
```

Comment:
Is this comment out of date? I can't find any other reference to `defaultBirthdate`.